### ISSUE TITLE
Remove formatting tools from WYSIWYG editors

### ISSUE THREAD

**Issue by paulstroot** [ocupop/comission-application#144]
Waiting on confirmation on Bill as to whether we disallow ALL formatting...

**Comment by ZiggyAtl** [comment]
@paulstroot please keep in the trimmed down formatting options that you have in there. One added piece of information, in case it helps you, when I tried to Add Art a short while ago, it let me add text in the Description section. However on the older items that are already in the Manage Art section, I can't select or change the text in that Description box.

**Comment by paulstroot** [comment]
Thanks for the details. For the styling options in the WYSIWYG editor,  I think this is ready for review. We can discuss the issue you're having with entering text for the description in another issue.
